### DISCUSSION TITLE
I am running the project on my windows10 local - but not working now.

### DISCUSSION THREAD

**Discussion by Wuffle1125** [kodadot/nft-gallery#5728] [General]
I setup pnpm and run the project using pnpm. After mixing, this error is occured.
![error](https://user-images.githubusercontent.com/17731477/232937822-62950d2b-c6cc-4a2f-a3d9-56af3a8fc0c7.png)

If there is anyone to know this error, plz let me know.
Thank you!

**Comment by damskyftw** [comment]
It appears that you are encountering a parsing error in your JavaScript code. The error message indicates that there are unexpected tokens found in the code, which suggests that there may be a syntax error or an incorrect configuration of plugins or loaders.

A common solution for resolving parsing errors with Webpack is to ensure that the correct loaders or plugins are installed, and the Webpack configuration is set up properly. Here are some steps you can take to troubleshoot the issue:

1. Check if you have the correct loaders installed, such as babel-loader for JavaScript or ts-loader for TypeScript, and ensure they are in the correct version.
2. Verify that your Webpack configuration is set up correctly. Make sure you don't have duplicate module properties in the config object or any other errors in your configuration.
3. If you are using any third-party libraries, ensure that you have properly transpiled the modules in your Webpack configuration.

**For more specific guidance, please refer to these resources:**

- Stack Overflow [Module parse failed: Unexpected token](https://stackoverflow.com/questions/52268320/module-parse-failed-unexpected-token)
- GitHub [Module parse failed: Unexpected token. You may need an appropriate...](https://github.com/akveo/react-native-ui-kitten/issues/996)
- GitHub [React - Module parse failed: You may need an appropriate loader to...](https://github.com/babel/babel-loader/issues/173)


In your case, you might need to add a loader to handle the .mjs files. Double-check your Webpack configuration to ensure the appropriate loader is set up for handling these files.
